docs/using-kontena/volumes.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Volume Management
+---
+
+# Volumes
+
+Kontena provides capability to manage persistent data using volumes.
+
+Volumes are supported as first-class citizens in both Kontena Master API and on stack yaml definitions.
+
+Currently Kontena does not automatically configure any volume drivers within the grid so user must configure those themselves.
+
+
+## Managing volumes with CLI
+
+User can manage volumes with Kontena CLI with `kontena volume xyz` commands
+
+### Creating volumes
+
+`kontena volume create --driver rexray --scope instance my-volume`
+
+See volume [scopes](#volume-scopes) for details on the different scopes.
+
+### Listing volumes
+
+`kontena volume ls`
+
+### Deleting volumes
+
+`kontena volume rm my-volume`
+
+Deleting a volume that is still in use in any of the services will fail.
```

Comment:
Deleting a volume that is still in use by any Kontena services will fail. Deleting an unused volume will remove the underlying Docker volumes from the host nodes.